### ISSUE TITLE
[Bug]: Not able to set Trees item line height

### ISSUE THREAD

**Issue by NKoepf** [mallowigi/material-theme-issues#483]
### Describe the bug

I do have the free version installed, and before the update to 9.1. I was able to set the tree line height, so my project tree was more compact. Now this option is not selectable. Is this a bug or is it now a premium feature? Other premium features are marked as such.

### To Reproduce

Open the intellij settings and navigate to Appearance & Behaviour > Material Theme UI > Trees

![Bildschirmfoto 2024-04-05 um 14 35 16](https://github.com/mallowigi/material-theme-issues/assets/38218138/b95bf436-70f6-4982-a7bd-f6c36ac1aecb)

### Expected behavior

To be able to set custom item line height

### Material Theme Version

v9.1

### IDE

IntelliJ IDEA

### System Info

```shell
IntelliJ IDEA 2024.1 (Ultimate Edition)
Build #IU-241.14494.240, built on March 28, 202
Runtime version: 17.0.10+8-b1207.12 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.

Macbook Pro 2019 Intel i9
```

**Comment by mallowigi** [comment]
I see, this was available in the free plan before, I thought all Trees features needed a premium purchase. My bad, I'll reopen it for the next hotfix.

**Comment by Alecto** [comment]
the same for me

**Comment by mallowigi** [comment]
[Material Theme UI-9.2.0.zip](https://github.com/mallowigi/material-theme-issues/files/14889233/Material.Theme.UI-9.2.0.zip)

Upcoming version. I will not deploy it at the moment, in order to fix more bugs, but this should do the trick for this particular issue.

**Comment by NKoepf** [comment]
unfortunately the provided 9.2.0 version does not fix the issue for me. I have installed it but the trees options for spacing are still not selectable for me

![Bildschirmfoto 2024-04-08 um 08 50 06](https://github.com/mallowigi/material-theme-issues/assets/38218138/4c362dda-13aa-4af9-984c-940f03d76383)

**Comment by AndKiel** [comment]
This shouldn't be closed because the issue is not fixed at all. :/

Even though the checkbox can be enabled, changing the number and applying does nothing - when settings are closed and reopened again, the value goes back to 28.

**Comment by mallowigi** [comment]
are you on the latest version?

**Comment by AndKiel** [comment]
> are you on the latest version?

Yes, the plugins list shows version 9.2.0 which is currently the latest.

**Comment by mallowigi** [comment]
That's weird, I checked to see and it should work. Are you using the free plan?

**Comment by NKoepf** [comment]
Same for me, as I wrote above. I am using the free tier

**Comment by Alecto** [comment]
9.2.0, free plan - trees line height doesnt work.
not fixed

**Comment by mallowigi** [comment]
I see. The checkbox works but the value is still only open for premium users only. How unfortunate.

I'll issue another hotfix later that week, for now I'm still waiting for other issues to come up in order to fix them all together. Thank you for your patience.

**Comment by ALexanderMarginal** [comment]
I have the same issue with free and trial versions

**Comment by Alecto** [comment]
in 9.2.1 - the problem with trees not resolved

**Comment by mallowigi** [comment]
Can you show a video showing the problem? I definitely made sure it's working for the free plan...

**Comment by Alecto** [comment]
@mallowigi 

I started recording a video to show the problem, and during that process, I realized that the "Toggle Compact Item Line Height" option was enabled. I tried disabling and re-enabling it, and after doing so, the height issue was resolved. It seems that toggling this setting somehow fixed the problem. Thank you for prompting me to double-check!

![image](https://github.com/mallowigi/material-theme-issues/assets/8464662/e8f15f81-9b2d-436f-82a4-e8c3084dd90c)

![image](https://github.com/mallowigi/material-theme-issues/assets/8464662/abc2be0c-dd8b-462b-98a7-e3f921b8e8bd)
